Sources/_NIOBase64/Base64.swift

```diff
@@ -35,7 +35,7 @@ public enum Base64Error: Error {
 }
 
 @usableFromInline
-internal struct Base64 {
+internal enum Base64: Sendable {
```

Comment:
Do we need this to be `Sendable` even if it is just a namespace?

Reply:
Yes. Because it's `@usableFromInline`, `-require-explicit-sendable` applies to it, and that flag does not notice that you can't actually form a value to it.

Reply:
More broadly, `-require-explicit-sendable` forces our hand for everything. "Need" doesn't really come into it: we have to mark anything it wants us to mark.